src/common/Logging/MacAddressHash/MACInformationProvider.cs

```diff
@@ -69,7 +68,7 @@ internal static class Mono
         /// Check if there is a persisted value otherwise calculates and persist a new one
         /// </summary>
         /// <returns>The hash of the mac address</returns>
-        private string GetMacAddressHash()
+        public string GetMacAddressHash()

```

Comment:
@Eneuman The reason for making this public is for UT's ?

Reply:
@hsubramanianaks Yeah, it's for making it public for UT. I tried using `protected` but it wasn't enought.